cpp/modmesh/view/RManager.cpp

```diff
@@ -366,6 +367,10 @@ std::function<void()> RManager::createCameraMovementItemHandler(const std::funct
         {
             if (controllerType == CameraControllerType::Orbit)
             {
+                constexpr float orbitRotationSpeed = 5.0f;
+
+                input.rxAxisValue *= orbitRotationSpeed;
+                input.ryAxisValue *= orbitRotationSpeed;
```

Comment:
For orbit camera value is not 1.0f (as in FPS camera), because rotation is very small.

Reply:
We will need to experience this later after merging.  It's great that you are starting to fine-tune.